GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -0,0 +1,49 @@
+import Quick
+import Nimble
+import GuildedRoseLLC
+import XCTest
+
+class ItemViewControllerSpec: QuickSpec {
+    
+    override func spec() {
+        
+        describe("Loading the view") {
+            var controller: GuildedRoseLLC.ItemsViewController!
+            
+            beforeEach {
+                controller = GuildedRoseLLC.ItemsViewController()
+                let itemCollectionView = UICollectionView(frame: CGRect(x: 0, y: 0, width: 0, height: 0), collectionViewLayout: UICollectionViewLayout())
+                controller.itemCollectionView = itemCollectionView
+                controller.greeting = UILabel()
+                controller.noItemsLabel = UILabel()
+            }
+            
+            describe("viewDidLoad") {
+                it("creates a data source") {
+                    controller.viewDidLoad()
+                    
+                    expect(controller.itemCollectionView.dataSource).notTo(beNil())
+                }
+                
+                it("sets the data source to an empty list") {
+                    controller.viewDidLoad()
+                    
+                    let dataSource = controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource
+                    
+                    expect(dataSource?.items).to(equal([]))
+                }
+            }
+            
+            describe("viewWillAppear") {
+                it("sets the data source to a static item list") {
```

Comment:
Is this correct? `viewDidLoad` is what is setting the data source. `viewWillAppear` is showing the items.

I would expect to see tests verifying that the two elements are hidden / shown correctly.